compiler/luci/pass/src/QuantizeActivation.cpp

```diff
@@ -185,7 +181,7 @@ void QuantizeConstInputActivation::visit(luci::CircleNode *node)
   {                                                             \
     auto input = node->INPUT_NAME();                            \
     auto const_node = dynamic_cast<luci::CircleConst *>(input); \
-    if (const_node && !is_quantized(const_node))                \
+    if (const_node && is_fp32(const_node))                      \
```

Comment:
To reviewers:

Previous codes attempted to quantize non-fp32 nodes, which can raise an error. This PR resolves it.